CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 - Renamed `SetHiveConfValues` in `com.klarna.hiverunner.examples.junit4` to `SetHiveConfValuesTest`.
 - Updated `surefire-version-plugin` from `2.21.0` to `2.22.0`.
 - Updated `junit.jupiter.version` from `5.6.0` to `5.7.0`.
+- Initialised `scriptsUnderTest` variable in `HiveRunnerExtension` as `null` instead of en empty list.
```

Comment:
This won't mean much to end users, instead you should talk about what you've actually fixed (and put it in a `Fixed` section) - i.e. that the `files` thing for the Extension now actually works etc.